bin/__snapshots__/verifyExports.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Index exports,   if a EDS component was recently added or removed, fix index then update snapshots 1`] = `
+Array [
+  "optionbutton",
```

Comment:
shouldn't there be more components in here?

Reply:
Not sure about my approach here but this is basically leftover things, exports from index that dont actually have a .js file associated with it. But I see names in here that DO have .js files so something is broken.